lib/qif/writer.rb

```diff
@@ -61,11 +62,17 @@ def initialize(io, type = 'Bank', format = 'dd/mm/yyyy')
     
     # Add a transaction for writing
     def <<(transaction)
-      @transactions << transaction
+      case transaction.class.to_s
```

Comment:
Yes. would be better if it raised an exception then silently handling more than one account.
On that not should 'write' fail if there is no account?

Im not really sure how these files work im just going off the code and comments.